ViAn/utility.cpp

```diff
@@ -152,3 +152,34 @@ cv::Rect Utility::from_qrect(QRect rect) {
 QRect Utility::from_cvrect(cv::Rect rect) {
     return QRect(from_cvpoint(rect.tl()), from_cvpoint(rect.br()));
 }
+
+/**
+ * @brief Utility::rotate
+ * Rotate the given point by the given rotation
+ * @param pos           : Point to be rotated
+ * @param rotation      : Wanted rotation
+ * @param width         : Width of frame
+ * @param height        : Height of frame
+ * @return              : Rotated point
+ */
+QPoint Utility::rotate(QPoint pos, int rotation, int width, int height) {
+    const double DEGREE_TO_RADIAN_FACTOR = M_PI / 180;
```

Comment:
Should be assigned in the Utility namespace instead of the function so it isn't reassigned at every call